core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/ExpansionGreedyRegionPlanGenerator.scala

```diff
@@ -315,7 +317,16 @@ class ExpansionGreedyRegionPlanGenerator(
                     1
                   }
 
-                physicalOp.id -> (0 until workerCount).map(_ => WorkerConfig()).toList
+                physicalOp.id -> (0 until workerCount)
+                  .map(idx => {
+                    val workerId = VirtualIdentityUtils
+                      .createWorkerIdentity(physicalOp.executionId, physicalOp.id, idx)
```

Comment:
use PhysicalOp.identifier() function instead? I want to avoid using `createWorkerIdentity` multiple times. It's better to create id from one place.

Reply:
Do you mean `PhysicalOp.getWorkerIds`? The worker IDs are assigned after the creation of worker configs. But I need them before the creation.

Reply:
I don't see an easy way to get the worker ids right now. Will keep the current logic and fix it later.